images/dns-controller-builder/Dockerfile

```diff
@@ -12,16 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM gcr.io/google_containers/debian-base-amd64:0.1
```

Comment:
Finally had a chance to test, and this image does not have bash installed.  The `onbuild.sh` does not run.  Can you please test and ensure this runs in a cluster?